sympy/integrals/intpoly.py

```diff
@@ -1025,20 +1025,20 @@ def point_sort(poly, normal=None, clockwise=True):
                         sum(map(lambda vertex: vertex.z, pts)) / n)
 
     def compare(a, b):
-        if a.x - center.x >= S.Zero and b.x - center.x < S.Zero:
+        if a.x - center.x >= 0 and b.x - center.x < 0:
```

Comment:
If you make this change you force sympy to sympify the value. Why did you make these changes?

Reply:
There was a period of time where I was under the impression that using numbers was actually faster in relations. Let me double-check if it is true here.

Reply:
Not true here. Will revert.

Reply:
This was the observation I did:

```
%timeit 0 < y
10.8 µs ± 970 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

%timeit S.Zero < y
29.7 µs ± 1.22 µs per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

but it cannot be applied here.

Reply:
I would usually prefer to use `a < b` then `a - b < 0` (for clarity rather than performance, or possibly to handle infinities)

Reply:
Yeah, but now the purpose was performance. :-) (Without reducing readability.) Not sure that the total result is very striking in terms of performance though... But at least a few things should be better.